docker/tester-exec.sh

```diff
@@ -73,7 +73,7 @@ testimg() {
   # Execute with +e to make sure that possibly created log files get
   # renamed, archived, published even when AFT or some of renaming fails
   set +e
-  daft ${DEVICE} ${FILENAME} --record
+  daft ${DEVICE} ${FILENAME} --record --emulateusb
```

Comment:
It's done using libcomposite module and Beaglebone USB OTG port. We have already been using this functionality with DUT "support image". Currently we boot the support image with DUT and then using it flash the image to be tested to internal memory or sd card. The script that does it is here: https://github.com/skuusela/DAFT/blob/ac26176caa803e56941f95f1db615f8675153282/testing_harness_image_extras/initialize_testing_harness/start_libcomposite